stack.yaml

```diff
@@ -1 +1 @@
-resolver: lts-21.9
+resolver: lts-21.25
```

Comment:
question: why not directly use `lts-22.4`? It is stable and we do support the corresponding GHC version. Is there any reason not to use it here?

Reply:
No good reason. My mind is still stuck on "9.4.8 is currently marked as recommended in ghcup" but might as well go with your suggestion.

Reply:
Oh ok. I'm fine either way actually :slightly_smiling_face: